HW3/P5/label_regions.cl

```diff
@@ -87,13 +133,21 @@ propagate_labels(__global __read_write int *labels,
         // We set new_label to the value of old_label, but you will need
         // to adjust this for correctness.
         new_label = old_label;
```

Comment:
After parts 2 and 4, this should now be new_label = buffer[buf_y * buf_w + buf_x].